examples/Viewer.tsx

```diff
@@ -725,6 +725,29 @@ class Viewer extends React.Component<{}, ViewerState> {
                 >
                     Focus Mode
                 </button>
+                <br />
+                <button onClick={() => simulariumController.getMetrics(netConnectionSettings)}>
+                    Get available metrics
+                </button>
+                <button
+                    onClick={() =>
+                        simulariumController.getPlotData(
+                            netConnectionSettings,
+                            [
+                                {
```

Comment:
maybe include a comment that these hardcoded metric IDs will eventually be chosen by the user based on results of the getMetrics() call